app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -231,10 +234,55 @@ public QuadTree providesQuadTres() {
     @Provides
     @Singleton
     public AppDatabase provideAppDataBase() {
-        appDatabase=Room.databaseBuilder(applicationContext, AppDatabase.class, "commons_room.db").build();
+        Builder<AppDatabase> appDatabaseBuilder = Room
+            .databaseBuilder(applicationContext, AppDatabase.class, "commons_room.db");
+        appDatabaseBuilder.addMigrations(MIGRATION_1_2);
+        appDatabase = appDatabaseBuilder.build();
         return appDatabase;
     }
 
+    static final Migration MIGRATION_1_2 = new Migration(1, 2) {
+        @Override
+        public void migrate(SupportSQLiteDatabase database) {
+            database.execSQL("DROP TABLE `contribution`");
+            database.execSQL("CREATE TABLE IF NOT EXISTS `contribution` ("
```

Comment:
@ashishkumar468 Do we need to specify all the columns manually? Does room not pick the columns based on the model class. 

PS: I am not very familiar with how room works.

Reply:
Yes it does, but for migration, we have to let it know exactly what it's supposed to do, as you can see we only have the DB as the parameter to work upon